src/nanoFramework.Graphics/Graphics/Core/Support/Jpeg/jcmarker.c

```
@@ -584,16 +584,16 @@ write_scan_header(j_compress_ptr cinfo)
                 /* Progressive mode: only DC or only AC tables are used in one scan */
                 if (cinfo->Ss == 0) {
                     if (cinfo->Ah == 0)   /* DC needs no table for refinement scan */
-                        emit_dht(cinfo, compptr->dc_tbl_no, FALSE);
+                        emit_dht(cinfo, compptr->dc_tbl_no, false;
                 }
                 else {
-                    emit_dht(cinfo, compptr->ac_tbl_no, TRUE);
+                    emit_dht(cinfo, compptr->ac_tbl_no, true;
                 }
             }
             else {
```

Clang (14.0.6)

[note] else branch starts here

(clang)